aspnetcore/fundamentals/logging/index.md

```diff
@@ -132,7 +132,41 @@ In the preceding sample:
 
 ## Set log level by command line, environment variables, and other configuration
 
-Log level can be set by any of the [configuration providers](xref:fundamentals/configuration/index). For information on configuring the logging providers using the command line, environment variables, other file formats, and more, see <xref:fundamentals/configuration/index>.
+Log level can be set by any of the [configuration providers](xref:fundamentals/configuration/index). 
+
+[!INCLUDE[](~/includes/environmentVarableColon.md)]
+
+The following commands:
+
+* Sets the environment key and value of the `Logging:LogLevel:Microsoft:Information` on Windows.
+* Tests the settings when using an app created with the ASP.NET Core web application templates. The `dotnet run` command must be run in the project directory after using `set`.
+
+```cmd
+set Logging__LogLevel__Microsoft=Information
+dotnet run
+```
+
+The preceding environment setting:
+
+* Is only set in processes launched from the command window they were set in.
+* Won't be read by browsers launched with Visual Studio.
+
+The following [setx](/windows-server/administration/windows-commands/setx) command can be used to set the environment keys and values on Windows. Unlike `set`, `setx` settings are persisted. `/M` sets the variable in the system environment. If the `/M` switch isn't used, a user environment variable is set.
+
+```cmd
+set Logging__LogLevel__Microsoft=Information /M
+```
+
+On [Azure App Service](https://azure.microsoft.com/services/app-service/), select **New application setting** on the **Settings > Configuration** page. Azure App Service application settings are:
+
+* Encrypted at rest and transmitted over an encrypted channel.
+* Exposed as environment variables.
+
+For more information, see [Azure Apps: Override app configuration using the Azure Portal](xref:host-and-deploy/azure-apps/index#override-app-configuration-using-the-azure-portal).
+
+See [environment variables](xref:fundamentals/configuration/index#environment-variables) for more information.
```

Comment:
```suggestion
For more information on setting ASP.NET Core configuration using environment variables, see [environment variables](xref:fundamentals/configuration/index#environment-variables).
```

I'm not 100% on the wording I've suggested here. I'm trying to give some context on what the "more information" is about because there's another "more information" link directly above.